### PR TITLE
update netdata to v1.23.2

### PR DESCRIPTION
**Pull request by zheshifandian**
PKG_VERSION:=1.23.2

Maintainer: me / @\<github-user> (find it by checking history of the package Makefile)
Compile tested: (put here arch, model, OpenWrt version)
Run tested: (put here arch, model, OpenWrt version, tests done)

Description:

### CODE REVIEW

File: admin/netdata/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ PKG_CPE_ID:=cpe:/a:my-netdata:netdata
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/netdata/netdata/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=6efd785eab82f98892b4b4017cadfa4ce1688985915499bc75f2f888765a3446
+PKG_HASH:=761aec15901e09f963361752c6e5b5cb723e342a3c5bf6b0624067fceb2ccdfd
```
Comment (BKPepe) [inline admin/netdata/Makefile:21]:
```
+ curl -f --connect-timeout 20 --retry 5 --location --insecure https://codeload.github.com/netdata/netdata/tar.gz/v1.23.2?/netdata-v1.23.2.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 3749k    0 3749k    0     0  2417k      0 --:--:--  0:00:01 --:--:-- 2415k
Hash of the downloaded file does not match (file: b6dd13292e0b1fb4137b1f7c0dc3d7c5e8a1bf408b82c2b8394f3751800e0eb5, requested: 761aec15901e09f963361752c6e5b5cb723e342a3c5bf6b0624067fceb2ccdfd) - deleting download.
```

File: admin/netdata/Makefile [hunk 2]
```diff
@@ -52,6 +52,7 @@ CONFIGURE_ARGS += \
 	--with-math \
 	--disable-x86-sse \
 	--enable-lto \
+        --disable-cloud\
```
Comment (BKPepe) [inline admin/netdata/Makefile:55]: Why move it here?

File: admin/netdata/Makefile [hunk 3]
```diff
@@ -60,8 +61,7 @@ CONFIGURE_ARGS += \
 	--disable-plugin-cups \
 	--disable-plugin-xenstat \
 	--disable-backend-prometheus-remote-write \
-	--disable-unit-tests \
-	--disable-cloud
+	--enable-jsonc
```
Comment (BKPepe) [inline admin/netdata/Makefile:64]: It is no longer optional. It is not mention in the commit message.

File: admin/netdata/Makefile [hunk 4]
```diff
@@ -76,6 +76,9 @@ define Package/netdata/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/netdata $(1)/usr/lib
 	$(CP) $(1)/usr/lib/netdata/conf.d/fping.conf $(1)/etc
 	$(CP) $(1)/usr/lib/netdata/conf.d/health_alarm_notify.conf $(1)/etc
+	rm -rf $(1)/usr/lib/netdata/python.d/python_modules/pyyaml2
+	rm -rf $(1)/usr/lib/netdata/python.d/python_modules/pyyaml3
+	rm -rf $(1)/usr/lib/netdata/python.d/python_modules/urllib3
```
Comment (BKPepe) [inline admin/netdata/Makefile:81]: Those were removed earlier. Why they are coming back?

File: admin/netdata/Makefile [hunk 5]
```diff
@@ -88,7 +91,6 @@ define Package/netdata/install
 	rm $(1)/usr/share/netdata/web/images/*.png
 	rm $(1)/usr/share/netdata/web/images/*.gif
 	rm $(1)/usr/share/netdata/web/images/*.ico
-	rm -rf $(1)/usr/share/netdata/web/old
```
Comment (BKPepe) [inline admin/netdata/Makefile:91]: Why?

Comment (zheshifandian) [reply admin/netdata/Makefile:91]: My submission is modified from version 1.2.0
If you feel that some paragraphs are inappropriate, you can delete it

Comment (zheshifandian) [reply admin/netdata/Makefile]: Well, I have completed the modification according to your prompt

**Comment by diizzyy** [comment]
Superseded by #12914

**Pull request closed without merging**